openslides_backend/action/actions/motion/create.py

```diff
@@ -69,6 +69,15 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 raise ActionException(
                     "You can't give amendment_paragraph_$ in this context"
                 )
+        # if lead_motion and not has perm motion_can_manage
+        # use category_id and block_id from the lead_motion
+        if instance.get("lead_motion_id"):
```

Comment:
The comment above is right: This should only happens if there is a lead motion *and the user has not manage rights*. The second part is missing, so the test below is wrong.

Reply:
I use the perm check again and update the tests. 
If lead_motion_id is set and the user has _not_ the motion.can_manage right, then category_id and block_id are set from the lead motion.